operator/Makefile

```diff
@@ -7,6 +7,14 @@
 # dropped (e.g. 3.0.61.1 -> 3.61.1) and development version ".x" is changed to ".0" (e.g. 3.0.61.x-123 -> 3.0.61.0-123).
 VERSION ?= $(shell $(MAKE) --quiet --no-print-directory -C .. tag | sed -E 's@^(([[:digit:]]+\.)+)x(-)?@\10\3@g' | sed -E 's@^3.0.([[:digit:]]+\.[[:digit:]]+)(-)?@3.\1\2@g')
 
+# The operator version used by ClusterServiceVersion object and related tooling (bundle, index). It is prefixed with "v".
+CSV_VERSION ?= v$(VERSION)
+# Operator index image tag to install from, used by deploy*-via-olm targets.
+INDEX_IMG_TAG ?= $(CSV_VERSION)
```

Comment:
Please move `INDEX_IMG_TAG` closer to `INDEX_IMG_BASE`. Also, I suggest revising description a bit
```
# INDEX_IMG_BASE and INDEX_IMG_TAG variables can be overridden when calling deploy*-via-olm targets, e.g. to deploy catalogs from IIB or other externally-built indexes.
```